erizo/src/erizo/rtp/RtpExtensionProcessor.h

```diff
@@ -47,7 +47,7 @@ class RtpExtensionProcessor{
 
  private:
   std::vector<ExtMap> ext_mappings_;
-  std::array<RTPExtensions, 10> ext_map_video_, ext_map_audio_;
+  std::array<RTPExtensions, 20> ext_map_video_, ext_map_audio_;
```

Comment:
I understand 10 was an arbitrary value but if we go ahead and change it, why not make it actually match the space for the Extension Ids? (1 - 14 according to https://tools.ietf.org/html/rfc5285) so, with `15` we would also cover 0.